test/unit/helpers/xhr/MFA_ENROLL_enrolledHotp.js

```diff
@@ -0,0 +1,92 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "testStateToken",
+    "expiresAt": "2015-06-15T21:06:04.794Z",
+    "status": "MFA_ENROLL",
+    "_embedded": {
+      "user": {
+        "id": "00uhncCcppZD2158x0g3",
+        "profile": {
+          "login": "administrator1@clouditude.net",
+          "firstName": "Add-Min",
+          "lastName": "O'Cloudy Tud",
+          "locale": "en_US",
+          "timeZone": "America\/Los_Angeles"
+        }
+      },
+      "factors": [{
+        "enrollment": "OPTIONAL",
+        "status": "NOT_SETUP",
+        "factorType": "question",
+        "provider": "OKTA",
+        "vendorName": "OKTA",
+        "_links": {
+          "questions": {
+            "href": "https:\/\/foo.com\/api\/v1\/users\/00uhncCcppZD2158x0g3\/factors\/questions",
+            "hints": {
+              "allow": [
+                "GET"
+              ]
+            }
+          },
+          "enroll": {
+            "href": "https:\/\/foo.com\/api\/v1\/authn\/factors",
+            "hints": {
+              "allow": [
+                "POST"
+              ]
+            }
+          }
+        }
+      }, {
+        "enrollment": "OPTIONAL",
+        "status": "ACTIVE",
+        "factorType": "token:hotp",
+        "provider": "CUSTOM",
+        "profiles": [{
+          "id": '123',
+          "name": 'Entrust',
+          "_embedded": {
+            "enrolledFactors": []
+          }
+        }, {
+          "id": '124',
+          "name": 'Entrust2',
+          "_embedded": {
+            "enrolledFactors": [{}]
```

Comment:
if this is enrolled, shouldn't this array have some data instead of an empty object?

Reply:
empty object still gives length as 1. Will put in some data jic.

Reply:
I dont have a good example of enrolledFactor object just put in name for now.